### PR TITLE
Boilerplate: bootstrap at 9bb96bc6c58c9608efc9f4db3e0ab5dda5287d93

### PR DESCRIPTION
**Pull request by 2uasimojo**
This PR comprises multiple commits:
- Generated PR bootstrapping boilerplate
- Mostly-Makefile fixups to accommodate the above
- Fixes for golangci-lint
- Regenerated CRDs
- Clean up go.sum

See the individual commit messages for details.

### CODE REVIEW

**Comment by jewzaam** [comment]
I'm good with the non-boilerplate dir changes, don't know if you want someone else to review besides me.

**Comment by 2uasimojo** [comment]
Thanks Naveen. It wouldn’t be a bad thing to get another pair of boilerplate eyes on it. @dustman9000 would you mind having a look?

File: go.sum [hunk 1]
```diff
@@ -950,6 +950,7 @@ rsc.io/letsencrypt v0.0.1/go.mod h1:buyQKZ6IXrRnB7TdkHP0RyEybLx18HHyOSoTyoOLqNY=
 sigs.k8s.io/controller-runtime v0.3.0/go.mod h1:Cw6PkEg0Sa7dAYovGT4R0tRkGhHXpYijwNxYhAnAZZk=
 sigs.k8s.io/controller-runtime v0.4.0 h1:wATM6/m+3w8lj8FXNaO6Fs/rq/vqoOjO1Q116Z9NPsg=
 sigs.k8s.io/controller-runtime v0.4.0/go.mod h1:ApC79lpY3PHW9xj/w9pj+lYkLgwAAUZwfXkME1Lajns=
```
Comment (dustman9000) [inline go.sum:952]: can you `go mod tidy` to remove unneeded refs?

Comment (2uasimojo) [reply go.sum]: Done.

**Pull request merged**